### ISSUE TITLE
CA1707 reported for throwaway underscore identifier

### ISSUE THREAD

**Issue by Zenexer** [dotnet/roslyn-analyzers#2440]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

2.9.2

#### Diagnostic ID

[CA1707](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1707-identifiers-should-not-contain-underscores?view=vs-2019)

#### Repro steps

``` csharp
using Microsoft.AspNetCore.Html;
using Microsoft.AspNetCore.Mvc.Rendering;

namespace Test
{
    public static class HtmlHelperExtension
    {
        public static IHtmlContent Example(this IHtmlHelper _)  // CA1707 here
            => new HtmlString("Hello, World!");
    }
}
```

#### Expected behavior

No warning: `_` is a throwaway identifier that suppresses IDE0060.  If I used something else, I'd get IDE0060.

#### Actual behavior

CA1707 reported for parameter `_`.
